### ISSUE TITLE
📝 Documentation: example `replies.yml`

### ISSUE THREAD

**Issue by Pinjasaur** [JoshuaKGoldberg/refined-saved-replies#52]
### Bug Report Checklist

- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/refined-saved-replies/issues?q=is%3Aissue+label%3A%22area%3A+documentation%22) and found none that matched my issue.

### Overview

An example and/or minimal spec for what the `replies.yml` file is _supposed_ to look like. Based on the source code, it looks like `body` and `name` keys are required? https://github.com/JoshuaKGoldberg/refined-saved-replies/blob/9f919b1553ccd608fde3cff1461144c096b276c4/src/content-script.ts#L80-L85

### Additional Info

(Also based on the `replies.yml` in this repo 🙂)

**Comment by Pinjasaur** [comment]
Ah, I see there's a bit 🤌 of overlap with #9.

**Comment by JoshuaKGoldberg** [comment]
Indeed, but you filed this one and I appreciate you filing it, so we can close that one as the dup 😄.

Recapping my hopes:

> The README.md is now pretty sparse. Let's add...
> 
> * an explanation of how to use the extension
> * a sample `.github/replies.yml`
> * a link to this repo's replies
> * a gif/video of someone using it

Feel free to just post part of that (say, just the sample code). We can file a followup for the rest.

**Comment by JoshuaKGoldberg** [comment]
@all-contributors please add @pinjasaur for docs.

> 🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).
> Not all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.
> ...and of course, thank you for contributing! 💙

**Comment by RubyM0226** [comment]
Hi! I'm a first-time contributor working through CodePath's AI301 open source course. I'd love to work on this. Would you be open to a PR? I'm planning to start with the sample replies.yml and a brief usage explanation in the README.

**Comment by JoshuaKGoldberg** [comment]
Sure. I haven't touched this project in a while, though, so you'll have to be pretty self-directed.
